src/utils/timing.js

```diff
@@ -2,27 +2,32 @@
 import type { TimingItemType } from '../types';
 
 const timingMap: { [string]: number } = {};
+const preciseTimingMap: { [string]: number } = {};
 const log: TimingItemType[] = [];
 
-const now = (): number =>
-  typeof performance !== 'undefined' ? performance.now() : Date.now() / 1000;
+const now = (): number => Date.now();
+
+const preciseNow = (): number =>
```

Comment:
A non terribe idea would have been to do that:
```js
const performanceNow = require('fbjs/lib/performanceNow');
```
Or copy-paste the very simple polyfill directly from:
https://github.com/facebook/fbjs/blob/master/packages/fbjs/src/performance/performanceNow.js

But scratch that as Meltdown or Spectre fixes decrease the resolution in browsers to 1-2msec. We should just use `Date.now()` and be mindful of the low precision.